src/main/resources/application.yaml

```diff
@@ -166,7 +166,6 @@ logging:
   level:
     uk.gov.hmcts.reform.authorisation.filters: DEBUG
     org.springframework.batch.core: WARN
-    org.apache.pdfbox: ERROR
```

Comment:
I'm not sure if clearing out this log for the pdfbox library will cause other unnecessary logs from PDFBox library to appear. As part of https://tools.hmcts.net/jira/browse/EM-6191, we had this set to ERROR level. Once more, one of the PI tasks is to just log the logs that are required in order to save money.

Reply:
I think it is just added for font warnings ?